alpine/Dockerfile

```
@@ -7,13 +7,16 @@ LABEL org.opencontainers.image.title="alpine-debug-container" \
       org.opencontainers.image.url="https://github.com/adfinis-sygroup/debug-container" \
       org.opencontainers.image.licenses="AGPL-3.0-or-later"
 
+ADD entrypoint.sh /entrypoint.sh
+
 RUN apk add --no-cache \ 
     bash \
     bind-tools \
     curl \
     jq \
     openssl \
     tcpdump \
-    wget
+    wget && \
+    chmod +x /entrypoint.sh
 
-ENTRYPOINT ["/bin/bash"]
+ENTRYPOINT ["/entrypoint.sh"]
```

Instead of using a custom entrypoint script we could also run `sleep inf`:
```suggestion
CMD ["/bin/sleep", "inf"]
```

---

:open_mouth: 

TIL: `sleep inf`

---

`sleep inf` is awesome, but sadly not working in BusyBox, because it's not in the POSIX standard but rather a special case of handling floating point in GNU and BSD. https://unix.stackexchange.com/a/128272